openedx_external_enrollments/tests/tests_backends.py

```diff
@@ -0,0 +1,13 @@
+"""This file contains all the necessary backend in a test scenario."""
+
+
+class ApiKeyHeaderPermissionIsAuthenticated:
+    """Test class for openedx.core.lib.api.permissions.ApiKeyHeaderPermissionIsAuthenticated"""
+
```

Comment:
pass? @andrey-canon just to be explicit.

Reply:
unnecessary-pass  or unless-pass I don't remember

Reply:
Unnecessary pass statement, but it has not explanation on why we should avoid the pass statement... anyway let's keep it as it is. @andrey-canon

Reply:
sorry I didn't know that you don't know about that pylint suggestion, but  please check this link https://github.com/PyCQA/pylint/issues/2208 in order to learn more about this.

Reply:
You said "unnecessary-pass or unless-pass" so I checked both, and "unless-pass" has not reference as a pylint messages, the only valid message is :unnecessary-pass. In addition you are not running quality testing, and the message is only geenrated in Pyhton versions > 3. @andrey-canon But I said, keep it as it is.

Reply:
Again my fault, I didn't say that in an specific way due to I thought you have a bigger knowledge of that and as suggestion you should try to understand a bigger scenario, there are some commented line that includes the quality test, that means that quality tests will be included and by the way if you doesn't have tests, it doesn't mean that you can write bad code